### PR TITLE
GEOMESA-2674 Update Camel to 2.24.1 for geomesa-stream

### PR DESCRIPTION
**Pull request by mforeman19**
-Tested with Apache Camel components Kafka and AWS Kinesis
Signed-off-by: Matthew Foreman <mforeman@ccri.com>

### CODE REVIEW

**Comment by mforeman19** [comment]
-Also I should add this was tested on geomesa 2.3.0 but, as far as I can tell, geomesa-stream has not received updates between these two versions. What do you think?

**Comment by mforeman19** [comment]
Tested with Camel 2.24.1, GeoMesa 2.4.0-SNAPSHOT, and GeoServer 2.15.2

**Comment by mforeman19** [comment]
Updated calculate-cqs.sh to disregard colors

File: build/cqs.tsv [hunk 1]
```diff
@@ -130,8 +130,8 @@ org.apache.arrow:arrow-format	0.10.0	compile
 org.apache.arrow:arrow-memory	0.10.0	compile
 org.apache.arrow:arrow-vector	0.10.0	compile
 org.apache.avro:avro	1.7.5	compile
-org.apache.camel:camel-core	2.15.1	compile
-org.apache.camel:camel-scala	2.15.1	compile
+org.apache.camel:camel-core	2.24.1	compile
+org.apache.camel:camel-scala	2.24.1	compile
```
Comment (elahrvivaz) [inline build/cqs.tsv]: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20478

Comment (elahrvivaz) [reply build/cqs.tsv]: approved

**Pull request merged**